<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="speedSetting">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="suffix">
     <string>r/min</string>
    </property>
    <property name="maximum">
     <number>540</number>
    </property>
    <property name="singleStep">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Speed :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>SerialPort Setup</string>
    </property>
    <widget class="QComboBox" name="CB_port">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="CB_baudrate">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CB_databit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CB_stopbit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CB_paritybit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>No Parity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd Parity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even Parity</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud Rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Bit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Bit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity Bit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_openSerial">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Serial</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Btn_SetSpeed">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Stop">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
